src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/config/AuditConfig.java

```diff
@@ -383,4 +383,16 @@ public String toString() {
                 .filter(settings::hasValue)
                 .collect(Collectors.toSet());
     }
+
+    public static final ImmutableSet<String> FIELD_PATHS = ImmutableSet.of(
+            "/enable",
+            "/audit/enable_rest", "/audit/disabled_rest_categories",
```

Comment:
Should this be "enable_rest" or "enable_rest_categories" ?

Reply:
It is `disabled_rest_categories` as it blacklists categories ignored. This field is now dynamically fetched via Jackson properties of that class.